test/uk/gov/hmrc/minorentityidentificationfrontend/services/ValidateTrustKnownFactsServiceSpec.scala

```diff
@@ -93,13 +93,67 @@ class ValidateTrustKnownFactsServiceSpec extends AnyWordSpec with Matchers with
         mockRetrieveTrustKnownFactsConnector.retrieveTrustKnownFacts(testSautr) returns Future.successful(None)
         mockStorageService.storeIdentifiersMatch(testJourneyId, DetailsNotFound) returns Future.successful(SuccessfullyStored)
 
-        val result = await(TestJourneyService.validateTrustKnownFacts(testJourneyId, testSautr, Some("AB0 0AA")))
+        val result = await(TestJourneyService.validateTrustKnownFacts(journeyId = testJourneyId,
+          optSaUtr = Some(testSautr),
+          optSaPostcode = Some("AB0 0AA"),
+          optCHRN = Some(testCharityHMRCReferenceNumber))
+        )
 
         result mustBe DetailsNotFound
 
         mockStorageService.storeIdentifiersMatch(testJourneyId, DetailsNotFound) was called
       }
     }
+
+    "return UnMatchableWithoutRetry" when {
+      "the user provides no Sautr but provides CHRN (in this case postcode does not matter)" in {
```

Comment:
With no sautr the user would never have an SA postcode so we don't need that test. A user cannot have both a CHRN and an SA Postcode